ehrapy/tools/feature_ranking/_feature_importances.py

```diff
@@ -47,8 +50,13 @@ def rank_features_supervised(
             for each feature individually. Defaults to 'standard'.
         percent_output: Set to True to output the feature importances as percentages. Note that information about positive or negative
             coefficients for regression models will be lost. Defaults to False.
+        logging: Set to False to disable logging. Defaults to True.
```

Comment:
```suggestion
        verbose: Set to False to disable logging. Defaults to True.
```
Is what we use in those cases usually